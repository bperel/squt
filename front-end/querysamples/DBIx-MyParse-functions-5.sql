SELECT 'a' REGEXP BINARY 'A'
