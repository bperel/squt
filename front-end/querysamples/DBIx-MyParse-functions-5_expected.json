{
  "Warning": {
    "No alias": {
      "regexp": ["function", "SELECT"]
    }
  },
  "Functions": {
    "1": {
      "to": "0",
      "Constants": {
        "A": "A"
      },
      "name": "cast_as_binary",
      "alias": "1"
    },
    "0": {
      "to": "OUTPUT",
      "Constants": {
        "a": "a"
      },
      "name": "regexp",
      "alias": "0"
    }
  }
}